indico/core/plugins/__init__.py

```diff
@@ -180,6 +183,10 @@ def translation_domain(self):
         path = self.translation_path
         return Domain(path) if path else NullDomain()
 
+    @property
```

Comment:
`@cached_property`?

Reply:
The manifest loader actually has its own cache, but yeah, we save ourselves the trouble of concatenating a few strings.